src/deployconfig.jl

```diff
@@ -241,11 +253,11 @@ function deploy_folder(cfg::Travis;
         @warn """
         Possible deploydocs() misconfiguration: main vs master
         Documenter's configured primary development branch (`devbranch`) is "master", but the
-        current branch (\$TRAVIS_BRANCH) is "main". This can happen because Documenter uses
-        GitHub's old default primary branch name as the default value for `devbranch`.
+        current branch (\$TRAVIS_BRANCH) is "main".
 
         If your primary development branch is 'main', you must explicitly pass `devbranch = "main"`
         to deploydocs.
+        Or remove the `devbranch` setting to fallback to the default which accepts both.
 
         See #1443 for more discussion: https://github.com/JuliaDocs/Documenter.jl/issues/1443
         """
```

Comment:
Maybe we can just delete this whole condition and warning message?
(and the other several copies of it for the other CI providers)

It seems like people are not going to do it by mistake anymore, with the new default?

Reply:
@mortenpi you added this warning in #1489
what do you think of just removing it?